### PR TITLE
[BUG] Fix errors in the Host isolation exception topic

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2980

Changed `type` field value from `endpoints` to `endpoint_host_isolation_exceptions` in the [Create host isolation exceptions container](https://security-docs_3001.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions-api.html#_create_host_isolation_exceptions_container) section:
- [Request body](https://security-docs_3001.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions-api.html#_request_body_37)
- [Example request](https://security-docs_3001.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions-api.html#_example_request_50)
- [Response payload](https://security-docs_3001.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions-api.html#_response_payload_54)

### CODE REVIEW

File: docs/management/api/_exceptions-api-reusable-content.asciidoc [hunk 1]
```diff
@@ -68,6 +68,23 @@
 |==============================================
 // end::create-exception-container-request-body[]
 
+// tag::create-host-isolation-exception-container-request-body[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`description` |String |Describes the {endpoint-artifact-name} container. |Yes
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
```
Comment (benironside) [inline docs/management/api/_exceptions-api-reusable-content.asciidoc:77]:
```suggestion
|`list_id` |String |Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
```

Comment (benironside) [reply docs/management/api/_exceptions-api-reusable-content.asciidoc:77]: Weirdly I couldn't see where this extra A appears in the text, but it seems extra 🤷

File: docs/management/api/_exceptions-api-reusable-content.asciidoc [hunk 2]
```diff
@@ -68,6 +68,23 @@
...
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The {endpoint-artifact-name} container's name. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
```
Comment (benironside) [inline docs/management/api/_exceptions-api-reusable-content.asciidoc:80]:
```suggestion
|`namespace_type` |String |Must be set to `agnostic`. |Yes
```
same as above, what am I missing?

File: docs/management/api/_exceptions-api-reusable-content.asciidoc [hunk 3]
```diff
@@ -68,6 +68,23 @@
...
+|`tags` |String[] |String array containing words and phrases to help categorize the
+{endpoint-artifact-name} container. |No
+|`type` |String a|Must be set to `endpoint_host_isolation_exceptions`. |Yes
```
Comment (benironside) [inline docs/management/api/_exceptions-api-reusable-content.asciidoc:83]:
```suggestion
|`type` |String |Must be set to `endpoint_host_isolation_exceptions`. |Yes
```
Same as above, please enlighten me 🤣

**Review by benironside — APPROVED** [review]
LGTM except one minor weird thing I was confused by. Thanks!

File: docs/management/api/_exceptions-api-reusable-content.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/management/api/_exceptions-api-reusable-content.asciidoc:77]: Yeah, this definitely looks like a minor typo. Good catch, thanks!

File: docs/management/api/_exceptions-api-reusable-content.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/management/api/_exceptions-api-reusable-content.asciidoc:83]: lol idk but good eye!

**Review by jmikell821 — APPROVED** [review]
LGTM

**Pull request merged**